gensim/models/base_any2vec.py

```diff
@@ -213,6 +214,9 @@ def _train_epoch(self, data_iterable, cur_epoch=0, total_examples=None,
             for _ in xrange(self.workers)
         ]
 
+        # Chain all input streams into one, because multistream training is not supported yet.
+        if multistream:
```

Comment:
Better to raise `warnings.warn` about lack of support (I mean explicitly to the user, not only comment)

Reply:
Hm, I don't think so:

We add new functionality, allowing users to use `multistream` mode. We added this functionality to the gensim core, so we are sure that it works correctly. Why should we __warn__ the user that something is wrong? Furthermore, user __can't do anything__ to get rid of this warning message while continuing using multistream mode.

Reply:
About **can't do anything** - not true, see https://docs.python.org/2/library/warnings.html#temporarily-suppressing-warnings (that's FYI), but I agree with your point in this case.